src/nupic/algorithms/BacktrackingTMCpp.hpp

```diff
@@ -35,16 +35,17 @@
 #include <vector>
 #include <string>
 
-using namespace std;
-using namespace nupic;
-using namespace nupic::algorithms::Cells4;
```

Comment:
I think this started the problem with namespaces. The using namespace was, and should not, be used outside of another namespace. here was included globally.

Reply:
good catch 👍